resources/views/layouts/app.blade.php

```diff
@@ -200,7 +200,7 @@ class="nav-link notifications-board-toggle"
                     / <a href="{{ route('legal.notice') }}">{{ __('menu.legal-notice') }}</a>
                 </span>
                         <span class="footer-nav-link">
-                    / <a href="{{ route('blog.all') }}">{{ __('menu.blog') }}</a>
+                    / <a href="https://blog.traewelling.de">{{ __('menu.blog') }}</a>
```

Comment:
Can you add a target=_blank or target=blog? So people don't get away from our page?